docs/modules/ROOT/pages/access-control.adoc

```diff
@@ -202,3 +134,132 @@ TIP: A recommended configuration is to grant both roles to a secure governance c
+
+This is accomplished with an xref:api:access.adoc#AccessManager[`AccessManager`] instance; a contract to store multiple smart contract permissions. Instead of managing each contract separately, it allows managing access from a single contract, making your protocol easier to audit and control.
+
+image::access-manager.svg[Access Manager]
+
+The manager allows granting permissions to a role, scoped by contract (called target) and https://docs.soliditylang.org/en/v0.8.20/abi-spec.html#function-selector[function selector] (called function). In this model, each role is allowed to call multiple functions in different contracts. Similarly, a role can be granted to multiple addresses, and a single address can be granted with multiple roles.
+
+image::access-manager-functions.svg[Access Manager functions]
+
+=== Using `AccessManager`
+
+Contracts includes xref:api:access.adoc#AccessManager[`AccessManager`], it can be deployed and used out of the box. It sets an initial admin in the constructor, who will be allowed to perform management operations.
+
+For a fresh new contract, you must implement the xref:api:access.adoc#AccessManaged[`AccessManaged`] contract provided along with the manager for matching permissions. This contract should define an Access Manager instance as its xref:api:access.adoc#AccessManaged-authority--[authority]. 
+
+Here's a simple example of an xref:tokens.adoc#ERC20[`ERC20`] token that defines a `mint` functionality that is restricted by an xref:api:access.adoc#AccessManager[`AccessManager`]:
+
+```solidity
+include::api:example$access-control/AccessManagedERC20MintBase.sol[]
+```
+
+NOTE: Make sure you fully understand how xref:api:access.adoc#AccessManager[`AccessManager`] works before using it or copy-pasting the examples from this guide.
+
+Once the managed contract has been deployed, it is now under manager's control, which will get the minter role assigned to an address and also allow such role to call the `mint` function.
+
+```javascript
+const MINTER = 42n; // Roles are uint64 (0 is reserved for the ADMIN_ROLE)
+
+// Allow the minter role to call the function selector
+// corresponding to the mint function
+await manager.setTargetFunctionRole(
+    myToken.address,
+    ['0x40c10f19'], // bytes4(keccak256('mint(address,uint256)'))
+    MINTER,
+    { from: initialAdmin }
+);
+```
+
+Even though each role has its own list of permissioned functions. Each address granted with a role (called member) will have an execution delay that will dictate how long the account should wait to execute a function allowed to its role. Delayed operations are possible by using the xref:api:access.adoc#AccessManager-schedule-address-bytes-uint48-[`schedule`] and xref:api:access.adoc#AccessManager-execute-address-bytes-[`execute`] functions.
+
+Additionally, each role granted to a member will have a granting delay before it takes effect. This is how you can set roles with a granting and execution delay:
```

Comment:
```suggestion
Additionally, roles can have a granting delay, that prevents for adding new members immediatly delays granted to a member will have a granting delay before it takes effect. The administrator of the Access Manager can set this grant delay as follows:
```

Reply:
The suggestion looks like there's an extra section not removed. I updated with a sentence similar to yours, let me know if it's ok